app/yarn.lock

```diff
@@ -5,7 +5,6 @@
 "@babel/cli@^7.0.0":
   version "7.4.3"
   resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.4.3.tgz#353048551306ff42e5855b788b6ccd9477289774"
-  integrity sha512-cbC5H9iTDV9H7sMxK5rUm18UbdVPNTPqgdzmQAkOUP3YLysgDWLZaysVAfylK49rgTlzL01a6tXyq9rCb3yLhQ==
```

Comment:
the massive negative diff on this file is quite worrying - I'm pretty confident that you used an old version of yarn for this (same thing happened with @tomrf1) I recommend reverting just this file and ensuring you've run `nvm use` in the `app` directory before then running `yarn` to install your newly added dependencies - I would say this is pretty important before merging

Reply:
OK will do, I'm pretty sure this is related to an older version of yarn and I'm happy to update as suggested!